.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.yml

```diff
@@ -18,6 +19,7 @@ body:
         - GTK+ app on Linux, BSD, etc.
         - Qt app on Linux, BSD, etc.
         - Qt app on Windows
+        - Qt app on macOS
```

Comment:
This is off-topic for this PR but I'll let it slide if you make the other change :smile_cat:

Reply:
:-)

Reply:
Why tho?
Is it something distributed elsewhere?

Reply:
.... yeah that is a fair point. I was using it as a friendly way to prompt Gary to use my copyediting suggestion, but no we're not distributing the Qt app on macOS. So having that option here might be confusing to most people.

Reply:
But users do build it.

Reply:
Maybe?

I'll be honest, I don't think I've ever experienced a Qt-app-on-macOS bug in the wild except for from you :smile:

Reply:
Fair enough. I'll take it out.